public/docs/ts/latest/guide/architecture.jade

```diff
@@ -163,27 +160,23 @@ figure
 
 :marked
   A _component_ controls a patch of screen called a *view*.
-  The shell at the application root with navigation links, a list of heroes, a hero editor ...
-  they're all views controlled by components.
+  The shell at the application root with navigation links, a list of heroes, and a hero editor are all views controlled by components.
 
   You define a component's application logic&mdash;what it does to support the view&mdash;inside a class.
   The class interacts with the view through an API of properties and methods.
 
   <a id="component-code"></a>
-  A `HeroListComponent`, for example, might have a `heroes` property that returns !{_an} !{_array} of heroes
-  that it acquired from a service.
-  It might have a `selectHero()` method that sets a `selectedHero` property when the user clicks to choose a hero from that list.
-  The component might be a class like this:
+  For example, this `HeroListComponent` has a `heroes` property that returns !{_an} !{_array} of heroes
+  that it acquires from a service.
+  It also has a `selectHero()` method that sets a `selectedHero` property when the user clicks to choose a hero from that list.
```

Comment:
Did you check whether it does indeed have this method? (The "might" made me think it didn't.)

Consider clarifying what "It" refers to.